test/evergreen.yml

```diff
@@ -224,22 +224,29 @@ functions:
         set -o verbose
 
         for i in $(seq ${times|1}); do
-          # Run the various combinations of args.  Let time and threads be random.
+          # Run the various combinations of args. Let time and threads be random. Add a
+          # timing stress to test_timestamp_abort every other run.
+          if [ $(( $i % 2 )) -eq 0 ]; then
+            tta_args=-s
```

Comment:
I guess variable name `tta_args` is not descriptive enough. There is no harm is using fuller name here.

Reply:
Changed the variable name to `test_timestamp_abort_args`.